src/Models/Campaign.php

```diff
@@ -255,7 +256,12 @@ public function getBounceRatioAttribute()
     public function getMergedContentAttribute(): ?string
     {
         if ($this->template_id) {
-            return str_replace(['{{content}}', '{{ content }}'], $this->content, $this->template->content);
+            return Blade::render(
+                $this->template->content,
+                [
+                    'content' => $this->content
```

Comment:
https://github.com/mettle/sendportal-core/pull/187#issuecomment-1590633349

```
return Blade::render(
                $this->template->content,
                [
                    'content' => $this->content,
                    // placeholders
                    'email' => '{!! $email !!}',
                    'first_name' => '{!! $first_name !!}',
                    'last_name' => '{!! $last_name !!}',
                    'unsubscribe_url' => '{!! $unsubscribe_url !!}',
                    'webview_url' => '{!! $webview_url !!}',
                ]
            );
```

Reply:
Thanks. I made a commit!

Reply:
I got this error.

```
Undefined constant "email"
```